### ISSUE TITLE
Encryption broken with Grub

### ISSUE THREAD

**Issue by h-2** [nightah/void-install#3]
I followed your instructions and the first reboot into the installed system went fine. But after I added an encrypted home dataset (root is still unencrypted!) and rebootet, GRUB could no longer identify the filesystem.
I think this is the issue: https://github.com/openzfs/zfs/issues/8810
most resources indicate that GRUB is just a bad choice combined with ZFS.... Would you recommend other bootloaders and could you add those to the tutorial? I tried installing rEFInd but this doesn't work because it requires VFAT...

I come from *BSD and my Linux knowledge is quite rusty. The whole process of installing Void seems quite complicat :crying_cat_face: Although your tutorial was helpful until now!

**Comment by tynanbe** [comment]
@h-2 this might be helpful:
https://github.com/zdykstra/zfsbootmenu/wiki

**Comment by h-2** [comment]
Yes, I have switched to Trident witch does all the right things by default.
